sources/packages/backend/workflow/test/2021-2022/fulltime-assessment/appeals/fulltime-assessment-2021-2022-PDPPD-status.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedFulltimeData,
+  executeFullTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-PDPPD-status.`, () => {
+  it("Should be true when student selected PD/PPD status through the application without any appeal.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+    // Act
+    const calculatedAssessment = await executeFullTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.calculatedDataPDPPDStatus).toBe(true);
+  });
+
+  it(
+    "Should be true when student did not select PD/PPD status from the application but requested a change " +
```

Comment:
Just a suggestion on the wording with little more business context. Should calculate the PD/PPD status to true when student did not choose to apply with disability funding in student application and then requested a change later with disability funding.